### PR TITLE
[ID] Update `Ranking Criteria`

### PR DESCRIPTION
**Pull request by mousesalt**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

Updated from:
https://github.com/ppy/osu-wiki/pull/6890
https://github.com/ppy/osu-wiki/pull/6951
https://github.com/ppy/osu-wiki/pull/6870
https://github.com/ppy/osu-wiki/pull/6840
https://github.com/ppy/osu-wiki/pull/6828

### CODE REVIEW

File: wiki/Ranking_Criteria/id.md [hunk 1]
```diff
@@ -13,6 +13,16 @@ Perubahan aturan dan pedoman dalam dokumen ini diusulkan dan dibahas di [forum R
 
 **Perlu diingat bahwa [kode etik dalam modding dan mapping](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping) dan [aturan konten musik](/wiki/Rules/Song_Content_Rules) adalah bagian dari ranking criteria dan berlaku untuk semua mode permainan.**
 
+## Alat-alat
+
+Kamu sangat direkomendasikan menggunakan [Mapset Verifier (MV)](https://github.com/Naxesss/MapsetVerifier) untuk membantu memeriksa Ranking Criteria. Verifikasi menggunakan [AiMod](/wiki/Client/Beatmap_editor/AiMod) dari klien lama (stable) sudah usang dan tidak dirawat, dan verifikasi beatmap di dalam klien baru (lazer) masih [dalam tahap pengembangan (WIP)](https://github.com/ppy/osu/issues/12091#issuecomment-878760791).
```
Comment (ZeroPyrozen) [inline wiki/Ranking_Criteria/id.md:18]:
```suggestion
Kamu sangat direkomendasikan menggunakan [Mapset Verifier (MV)](https://github.com/Naxesss/MapsetVerifier) untuk membantumu dalam memeriksa apakah beatmapmu sudah sesuai dengan Ranking Criteria. Verifikasi menggunakan [AiMod](/wiki/Client/Beatmap_editor/AiMod) dari klien lama (stable) sudah usang dan tidak dirawat, dan verifikasi beatmap di dalam klien baru (lazer) masih [dalam tahap pengembangan (WIP)](https://github.com/ppy/osu/issues/12091#issuecomment-878760791).
```

File: wiki/Ranking_Criteria/id.md [hunk 2]
```diff
@@ -13,6 +13,16 @@ Perubahan aturan dan pedoman dalam dokumen ini diusulkan dan dibahas di [forum R
...
+
+Hal-hal penting untuk dimengerti sebelum menggunakan Mapset Verifier (MV):
```
Comment (ZeroPyrozen) [inline wiki/Ranking_Criteria/id.md:20]:
```suggestion
Hal-hal yang perlu diperhatikan sebelum menggunakan Mapset Verifier (MV):
```

File: wiki/Ranking_Criteria/id.md [hunk 3]
```diff
@@ -13,6 +13,16 @@ Perubahan aturan dan pedoman dalam dokumen ini diusulkan dan dibahas di [forum R
...
+
+- Tidak ada alat yang sepenuhnya menggantikan keharusan untuk membaca atau memeriksa halaman ini secara manual.
+- Fokus tentang apa yang ditunjukkan dan jangan ikuti secara membabi buta.
+- Mereka ada untuk membantumu, tapi tidak untuk menggantikanmu.
```
Comment (ZeroPyrozen) [inline wiki/Ranking_Criteria/id.md:24]:
```suggestion
- Tidak ada alat yang sepenuhnya menggantikan keharusan untuk membaca atau memeriksa halaman ini secara manual.
- Fokus tentang apa yang ditunjukkan oleh alat tersebut dan jangan ikuti secara membabi buta.
- Mereka ada untuk membantumu, tetapi bukan untuk menggantikanmu.
```

**Review by ZeroPyrozen — DISMISSED** [review]

**Comment by Walavouchey** [comment]
forgot to remove outdated marks

**Comment by mousesalt** [comment]
thanks for reminder, removed

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**